text/0026-finality-signature-gossiping.org

```diff
@@ -0,0 +1,195 @@
+#+STARTUP: inlineimages
+
+* Finality Signatures For Secure Joining Node Synchronization
+:PROPERTIES:
+:CUSTOM_ID: finality-signatures-for-secure-joining-node-synchronization
+:END:
+
+** Summary
+:PROPERTIES:
+:CUSTOM_ID: summary
+:END:
+
+CEP PR: [[https://github.com/casperlabs/ceps/pull/0025][casperlabs/ceps#0025]]
+
+Nodes joining the network will use /finality signatures/ to securely
+synchronize with the validators. As a node synchronizes, it will get
+signatures of block hashes via the gossip network until it has > 33%
```

Comment:
We should probably use the fault tolerance as configured in the chainspec here.

Reply:
Here and in all places where 33% or 1/3 is mentioned, right?

Reply:
Fixed in https://github.com/CasperLabs/ceps/pull/26/commits/2196a7433bfa90b896e7271442df41760c6a4a73